src/tactic/ui/panel/collection_wdg.py

```diff
@@ -360,7 +365,44 @@ def execute(my):
             search_codes = [x.get_value("search_code") for x in items]
```

Comment:
the indentation is too much here 
execute(my): 

search_codes 
should line up on the left